### PR TITLE
C#/Java: Only display 1k most relevant results for ExternalApi telemetry queries.

### PR DESCRIPTION
**Pull request by michaelnebel**

### CODE REVIEW

**Comment by michaelnebel** [comment]
DCA run for C# looks fine. For Java there were some errors - most likely related to extraction problems and not the content of this PR.

File: java/ql/src/Telemetry/ExternalLibraryUsage.ql [hunk 1]
```diff
@@ -9,13 +9,28 @@
...
       not c.getFile() instanceof GeneratedFile and
       a.jarContainer() = jarname and
       not a.isUninteresting()
     )
+}
+
+private int getOrder(string jarname) {
+  jarname =
+    rank[result](string jar, int usages |
+      exists(ExternalApi api | jar = api.jarContainer()) and getRelevantUsages(jar, usages)
```
Comment (aschackmull) [inline java/ql/src/Telemetry/ExternalLibraryUsage.ql:25]:
```suggestion
      getRelevantUsages(jar, usages)
```

**Review by aschackmull — COMMENTED** [review]
A couple of superfluous lines, otherwise LGTM.

File: csharp/ql/src/Telemetry/ExternalLibraryUsage.ql [hunk 2]
```diff
@@ -10,12 +10,28 @@ private import csharp
...
       c = api.getACall() and
       api.getInfoPrefix() = info and
       not api.isUninteresting()
     )
+}
+
+private int getOrder(string info) {
+  info =
+    rank[result](string i, int usages |
+      exists(ExternalApi api | i = api.getInfoPrefix()) and
```
[the remark this answers is not in the document]
Comment (michaelnebel) [reply csharp/ql/src/Telemetry/ExternalLibraryUsage.ql:25]: Oh right! Thank you!

**Review by aschackmull — DISMISSED** [review]

**Review by aschackmull — APPROVED** [review]

**Comment by michaelnebel** [comment]
Ran DCA again. Everything still looks fine. Merging now.

**Pull request merged**